internal/adapters/rest/handlers/payment_request_create.go

```diff
@@ -0,0 +1,38 @@
+package handlers
+
+import (
+	"encoding/json"
+	"net/http"
+
+	"github.com/IgorSteps/easypark/internal/adapters/rest/models"
+	"github.com/sirupsen/logrus"
+)
+
+// PaymentRequestCreateHandler provides a REST Handler implementation to create payment requests and
+// implements http.Handler interface.
+type PaymentRequestCreateHandler struct {
```

Comment:
Let's call it `PaymentCreateHandler` - no need for `request` in there, afterall users are making payments